megamek/src/megamek/client/ui/swing/unitSelector/TWAdvancedSearchPanel.java

```diff
@@ -1563,6 +1566,7 @@ public boolean matches(String name) {
                     WeaponClass.MACHINE_GUN.matches(name);
             } else if (this == RAC) {
                 if (name.toLowerCase().contains("rotary")) {
+                    System.err.println(name);
```

Comment:
We try to avoid System.err.println logging - I would use LogManager.getLogger() if you really need to log this. Otherwise, let's just get rid of it.

Reply:
*facepalm* I meant to remove that.

Reply:
I removed in the latest commit.